gcapi/client.py

```diff
@@ -578,7 +578,7 @@ def _upload_files(self, *, files, **kwargs):
                 uploads.append(
                     (
                         yield from self.__org_api_meta.uploads.upload_fileobj(
-                            fileobj=f, filename=file.name
+                            fileobj=f, filename=f.name
```

Comment:
This is the actual fix, the rest is tests to cover both `str` and `Path` file uploads.